supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesFinalizationActor.scala

```diff
@@ -40,9 +39,7 @@ class JesFinalizationActor (override val workflowDescriptor: BackendWorkflowDesc
 
   private def deleteAuthenticationFile(): Future[Unit] = {
     if (jesConfiguration.needAuthFileUpload) {
-      val delete = () => Future(workflowPaths.gcsAuthFilePath.delete(false))
-
-      Retry.withRetry(delete, isFatal = isFatalJesException, isTransient = isTransientJesException)(context.system) map { _ => () }
+      Future(workflowPaths.gcsAuthFilePath.delete(false)) map { _ => () }
```

Comment:
I'm going to guess there's already a sufficient layer of retries in the GCS filesystem code?